spec/requests/client_visibility_spec.rb

```diff
@@ -5,28 +5,33 @@
 RSpec.describe ClientsController, type: :request do
   include_context 'visibility test context'
 
-  configs_variations = GrdaWarehouse::Config.available_cas_methods.values.product( # cas available method
-    [true, false], # project type override
-    GrdaWarehouse::Config.family_calculation_methods.values, # family calculation method
-    GrdaWarehouse::Config.available_release_durations, # release duration
-  )
+  configs_variations = []
+  GrdaWarehouse::Config.available_cas_methods.values.product( # cas available method
+    [true, false], # consent visible to all
+    [true, false], # expose coc code
+    GrdaWarehouse::Config.available_health_emergencies.values, # health emergency
+    GrdaWarehouse::Config.available_health_emergency_tracings.values, # health emergency tracing
+    [0, 1, 20, 65, 200], # health priority age
```

Comment:
I could be wrong, but 200 seems a big old.  Let's limit this a bit to just `0, 20, 65`

Reply:
Yea I was just instinctively testing for an edge case here.